week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -37,7 +61,16 @@ Write a function that:
 - numbers greater 100 must be replaced with 100
 */
 
-function formatPercentage(arr) {}
+function formatPercentage(arr) {
+  for(let i = 0; i < arr.length; i++){
+    arr[i] = Number(arr[i].toFixed(2));
+    if(arr[i] > 100)
+    arr[i] = 100;
+    arr[i] = arr[i] + "%";
+    console.log(arr[i]);
+     }
+     return arr
```

Comment:
Please, pay more attention to clear code formatting and indentation, it helps a lot no only to others but also to you  by making it easier to read and reason about especially when you will revisit your code in the future

Reply:
yes , you are right